introduction_to_amazon_algorithms/jumpstart-foundation-models/domain-adaption-finetuning-gpt-j-6b.ipynb

```diff
@@ -0,0 +1,782 @@
+{
```

Comment:
<p>!pip install --upgrade sagemaker --quiet</p><p> </p><p>I don't think this is needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>It is needed. We discussed this last time</p>